koku/masu/test/util/aws/test_common.py

```diff
@@ -344,8 +345,11 @@ def test_remove_files_not_in_set_from_s3_bucket(self):
                 removed = utils.remove_files_not_in_set_from_s3_bucket("request_id", s3_csv_path, "manifest_id")
                 self.assertEqual(removed, [])
 
+    @override_settings(ENABLE_PARQUET_PROCESSING=False)
     def test_copy_data_to_s3_bucket(self):
         """Test copy_data_to_s3_bucket."""
+        # TODO: needs review and remove ENABLE_PARQUET_PROCESSING=False
```

Comment:
Did we tackle these todos?

Reply:
yes, these are now cleaned out
or add `@skip` for  some tests